ROMFS/px4fmu_common/init.d-posix/airframes/1070_boat

```diff
@@ -17,6 +17,11 @@ param set-default GND_SPEED_TRIM 3
 param set-default GND_THR_CRUISE 0.85
 param set-default GND_THR_MAX 1
 param set-default GND_THR_MIN 0
+param set-default GND_RATE_P 0.08
+param set-default GND_RATE_D 0.02
+param set-default GND_RATE_FF 0.0
+param set-default GND_RATE_I 0.0
+param set-default GND_RATE_MAX 0.5
```

Comment:
Shouldn't this be GND_YAW_RATE for readability?  "ground rate" is very vague and could apply to all kinds of things, including being misinterpreted as speed IMO.

Reply:
@jasta Thanks for the suggestion, and I agree that it would be a better naming. However, by using `GND_YAW_RATE`, parameters becomes too long (for example `GND_YAW_RATE_IMAX` ) and is not possible to add as a parameter